feat: Add limitation about data scrubbing in breadcrumbs

olksdr opened the pull request:
This PR adds mentions about some limitations which data scrubbing  has in the breadcrumbs. 
Where the scrubbing only works for message and data.

untitaker commented on src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx:
```diff
@@ -60,3 +60,9 @@ If you’ve accidentally sent sensitive data to the server, you likely don't to
+
+The following limitations generally apply to all server-side data scrubbing.
+
+- It is currently not possible to avoid scrubbing data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) even when category name added to Safe Fields. Only the message and data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) are used for data scrubbing. And is custom JSON data is provided the fields will be scrubbed according to the configured rules and Safe Fields.
```
```suggestion
- It is currently not possible to avoid scrubbing data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) by adding the category name added to Safe Fields. Only the message and data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) are used for data scrubbing. And is custom JSON data is provided the fields will be scrubbed according to the configured rules and Safe Fields.
```

jjbayer commented on src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx:
```diff
@@ -60,3 +60,9 @@ If you’ve accidentally sent sensitive data to the server, you likely don't to
+
+The following limitations generally apply to all server-side data scrubbing.
+
+- It is currently not possible to avoid scrubbing data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) by adding the category name added to Safe Fields. Only the message and data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) are used for data scrubbing. And is custom JSON data is provided the fields will be scrubbed according to the configured rules and Safe Fields ...
```
```suggestion
- It is currently not possible to avoid scrubbing data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) by adding the category name added to Safe Fields. Only the message and data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) are used for data scrubbing. And if custom JSON data is provided the fields will be scrubbed according to the configured rules and Safe Fields.
```

imatwawana reviewed (commented):
I've made some edits to follow our style standards and some grammar fixes as well.

olksdr commented:
Thanks @imatwawana! I addressed your suggestions in [aeca951](https://github.com/getsentry/sentry-docs/pull/5449/commits/aeca951afb69540aa6579c08d67745f8c00a313b). 
Please, take another look.

imatwawana reviewed (approved):
Looks good!
